include/psen_scan_v2/ros_scanner_node.h

```diff
@@ -104,7 +107,16 @@ ROSScannerNodeT<S>::ROSScannerNodeT(ros::NodeHandle& nh,
 template <typename S>
 void ROSScannerNodeT<S>::laserScanCallback(const LaserScan& scan)
 {
-  pub_.publish(toLaserScanMsg(scan, prefix_, x_axis_rotation_));
+  const auto laserScanMsg = toLaserScanMsg(scan, prefix_, x_axis_rotation_);
+  PSENSCAN_INFO_THROTTLE(
```

Comment:
```suggestion
  PSENSCAN_INFO_ONCE(
```
Please see http://wiki.ros.org/roscpp/Overview/Logging for `*_ONCE` and add it to `PSEN_SCAN_*` macros, if it isn't available yet.

Reply:
I introduced the -ONCE macros again by using a very large period of 1e9 seconds, this is more than 30 years. It's a quick fix without having to develop and test new code. Do you see any difficulty with that?

Reply:
Resolved now...